src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/SplitButtonTests/SplitButtonPage.xaml.cs

```diff
@@ -0,0 +1,199 @@
+{
+	/// <summary>
+	/// An empty page that can be used on its own or navigated to within a Frame.
+	/// </summary>
+	[SampleControlInfo("SplitButton", "MUX_Test")]
+	public sealed partial class SplitButtonPage : Page
+    {
+		private int _clickCount = 0;
+		private int _flyoutOpenedCount = 0;
+		private int _flyoutClosedCount = 0;
+
+		public MyCommand TestExecuteCommand;
+		private int _commandExecuteCount = 0;
+
+		private Flyout _placementFlyout;
+
+		public SplitButtonPage()
+		{
+			this.InitializeComponent();
+
+			//TODO: SplitButtonTestApi.SimulateTouch = false;
+
+			TestExecuteCommand = new MyCommand(this);
+
+			_placementFlyout = new Flyout();
+			_placementFlyout.Placement = FlyoutPlacementMode.Bottom;
+			TextBlock textBlock = new TextBlock();
+			textBlock.Text = "Placement Flyout";
+			_placementFlyout.Content = textBlock;
+
+			ControlStateViewer.ControlType = TestSplitButton.GetType();
+			ControlStateViewer.States = new List<string>
+			{
+				"Normal",
+				"FlyoutOpen",
+				"TouchPressed",
+				"PrimaryPointerOver",
+				"PrimaryPressed",
+				"SecondaryPointerOver",
+				"SecondaryPressed",
+			};
+
+			ToggleControlStateViewer.ControlType = TestSplitButton.GetType();
+			ToggleControlStateViewer.States = new List<string>
+			{
+				"Checked",
+				"CheckedFlyoutOpen",
+				"CheckedTouchPressed",
+				"CheckedPrimaryPointerOver",
+				"CheckedPrimaryPressed",
+				"CheckedSecondaryPointerOver",
+				"CheckedSecondaryPressed",
+			};
+		}
+
+		private void TestSplitButton_Click(object sender, object e)
+		{
+			ClickCountTextBlock.Text = (++_clickCount).ToString();
+		}
+
+		private void TestSplitButtonFlyout_Opened(object sender, object e)
+		{
+			FlyoutOpenedCountTextBlock.Text = (++_flyoutOpenedCount).ToString();
+		}
+
+		private void TestSplitButtonFlyout_Closed(object sender, object e)
+		{
+			FlyoutClosedCountTextBlock.Text = (++_flyoutClosedCount).ToString();
+		}
+
+		private void SimulateTouchCheckBox_Checked(object sender, RoutedEventArgs e)
+		{
+			//TODO SplitButtonTestApi.SimulateTouch = true;
+		}
+
+		private void SimulateTouchCheckBox_Unchecked(object sender, RoutedEventArgs e)
+		{
+			//TODO SplitButtonTestApi.SimulateTouch = false;
+		}
+
+		private void EnableCheckBox_Checked(object sender, RoutedEventArgs e)
+		{
+			DisabledSplitButton.IsEnabled = true;
+		}
+
+		private void EnableCheckBox_Unchecked(object sender, RoutedEventArgs e)
+		{
+			DisabledSplitButton.IsEnabled = false;
+		}
+
+		private void CanExecuteCheckBox_Checked(object sender, RoutedEventArgs e)
+		{
+			if (TestExecuteCommand != null)
+			{
+				TestExecuteCommand.UpdateCanExecute(true);
+			}
+		}
+
+		private void CanExecuteCheckBox_Unchecked(object sender, RoutedEventArgs e)
+		{
+			if (TestExecuteCommand != null)
+			{
+				TestExecuteCommand.UpdateCanExecute(false);
+			}
+		}
+
+		public void CommandExecute()
+		{
+			ExecuteCountTextBlock.Text = (++_commandExecuteCount).ToString();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5246693)